source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -113,8 +113,8 @@ content: |
            final BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
            final BsonString masterKeyArn = new BsonString("<Master Key ARN>"); // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
            DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(new BsonDocument() {{
-             put("region", masterKeyRegion);
-             put("key", masterKeyArn);
+               put("region", masterKeyRegion);
```

Comment:
For BsonDocument, you can use `append` to avoid this ugliness

Reply:
Thanks, will fix.